### PR TITLE
Update docs for on-premise case, for guiding how to download from AR

### PR DESCRIPTION
**Pull request by 0405ysj**

### CODE REVIEW

**Review by jemoreira — APPROVED** [review]

File: scripts/on-premises/single-server/README.md [hunk 1]
```diff
@@ -1,30 +1,57 @@
-# Activate cloud orchestrator in the remote server
+# Activate cloud orchestrator at on-premise server
 
-This page describes how to run cloud orchestrator managing docker instances
-containing Host Orchestrator inside.
+This page describes how to run cloud orchestrator at on-premise server, which
+managing docker instances containing host orchestrator inside.
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 2]
```diff
@@ -1,30 +1,57 @@
...
 
-## Prepare docker image
+Noting that this is under development, some features may be broken yet. Please
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 3]
```diff
@@ -1,30 +1,57 @@
...
+let us know if you faced at any bugs.
 
-Please follow the docker part of
-[README.md](https://github.com/google/android-cuttlefish/blob/main/README.md#docker)
-in `google/android-cuttlefish` github repository, and check if the docker image
-`cuttlefish-orchestration` exists.
+## Try cloud orchestrator
+
+Currently we're hosting docker images and its configuration files into Artifact
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 4]
```diff
@@ -1,30 +1,57 @@
...
+Registry.
+Please execute commands below if you want to download and run cloud
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 5]
```diff
@@ -1,30 +1,57 @@
...
 ```bash
 ssh -D ${SOCKS5_PORT} -q -C -N ${USERNAME}@${CLOUD_ORCHESTRATOR_IPv4_ADDRESS}
 ```
-
 ## Use cloud orchestrator by cvdr
 
-Please check every configuration in
-`scripts/on-premises/single-server/cvdr.toml` is set well, and follow the steps
-at [cvdr.md](/docs/cvdr.md).
+Config file for `cvdr` is located at
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 6]
```diff
@@ -1,30 +1,57 @@
...
+[scripts/on-premises/single-server/cvdr.toml](cvdr.toml).
+Please follow the steps at [cvdr.md](/docs/cvdr.md), to know how to use `cvdr`.
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 7]
```diff
@@ -1,30 +1,57 @@
...
+
+## Manually build and run cloud orchestrator
+
+Config file for cloud orchestrator is at
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 8]
```diff
@@ -1,30 +1,57 @@
...
+[scripts/on-premises/single-server/conf.toml](conf.toml).
+Please follow the steps at [cloud_orchestrator.md](/docs/cloud_orchestrator.md),
+to know how to build and run cloud orchestrator.
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

File: scripts/on-premises/single-server/README.md [hunk 9]
```diff
@@ -1,30 +1,57 @@
...
+
+Also, you may need to prepare another docker image that containing host
```
[the remark this answers is not in the document]
Comment (0405ysj) [reply scripts/on-premises/single-server/README.md]: Done.

**Pull request merged**